### ISSUE TITLE
Results of regularized models on WordNet similarities

### ISSUE THREAD

**Issue by akutuzov** [uhh-lt/path2vec#18]
@m-dorgham can you add to the 'regularizers' tab in the [Google Spreadsheet](https://docs.google.com/spreadsheets/d/1esXh-eNz76_86PikQe9-FwkeEpMz7Of6rzeiSS6lrD8/edit#gid=224323705) the evaluation scores calculated on WordNet similarities?
What you have there now are the scores calculated either in the 'dynamic' setup (on the `simlex_original.tsv` test set) or in the 'static' setup (on the test sets from `simlex/simlex_synsets` subdirectory).

What I'm asking for now are the Spearman correlation values calculated on the test sets from the `simlex` subdirectory:

- `simlex_jcn_brown.tsv`
- `simlex_jcn_semcor.tsv`
- `simlex_lch.tsv`

It seems we should focus more on this metrics, since it is more closely related to our original aim, that is, approximating arbitrary graph distance metrics with dense vectors.

**Comment by alexanderpanchenko** [comment]
it seems that these are the right numbers - otherwise they would not be so high, or i misunderstand something. 

Mohammad, could you please report also number w/o the regularization please (for your run so that it is 100% comparable)?

**Comment by m-dorgham** [comment]
@alexanderpanchenko , @akutuzov  Here are the scores without the regularizer:
jcn-semcor: 0.9245
jcn-brown: 0.9013
lch: 0.9339

**Comment by alexanderpanchenko** [comment]
Thanks! This is consistent with other evaluations.

**Comment by akutuzov** [comment]
@m-dorgham Can you do the same for the `vector size` tab?

**Comment by m-dorgham** [comment]
@akutuzov Ok I will do.

**Comment by alexanderpanchenko** [comment]
@m-dorgham please also train models on these two extra datasets 

http://ls.hpc.uio.no/~andreku/data/shp_thresh01-near50.tsv.gz
http://ls.hpc.uio.no/~andreku/data/wup_thresh03-near50.tsv.gz

They correspond to two other similarity measures, like lch, and we need them as well for the table.

**Comment by m-dorgham** [comment]
@alexanderpanchenko Do you want me to train them on the best configurations of hyperparameters only, or for all combinations of hyperparameters?

**Comment by alexanderpanchenko** [comment]
Please start from the best and continue with other values (so that we can quickly report in the table something meaningful already).
